test/services/project-data-service.ts

```diff
@@ -244,4 +264,39 @@ describe("projectDataService", () => {
 			assert.deepEqual(dataPassedToWriteJson, { dependencies: {} });
 		});
 	});
+
+	describe("getAssetsStructure", () => {
+		it("does not fail when App_Resources/Android and App_Resources/iOS do not exist", async () => {
+			const defaultEmptyData: any = {};
+			defaultEmptyData[CLIENT_NAME_KEY_IN_PROJECT_FILE] = {};
+			const testInjector = createTestInjector(JSON.stringify(defaultEmptyData));
+			const fs = testInjector.resolve<IFileSystem>("fs");
+			fs.readJson = (filePath: string): any => {
+				if (basename(filePath) === AssetConstants.imageDefinitionsFileName) {
+					return { android: {}, ios: {} };
```

Comment:
In case when `imageDefinitionsFileName` file does not exist, an error will be thrown. I'm wondering if we should check this case.

Reply:
The file is part of CLI resources, so in case it does not exist, something terrible happened :smile_cat: